.github/workflows/build.yml

```diff
@@ -2,16 +2,11 @@ name: Build
 
 on: push
 
+permissions:
+  # Permission for checking out code
+  contents: read
+
 jobs:
-  gh-api-quota-check:
```

Comment:
Shouldn't need this anymore since I think it was put in place to help debug the issues with the Github Auth resource tests.